xml/operations-objectstorage-swift_container_sync.xml

```diff
@@ -357,17 +363,16 @@ Containers in policy "erasure-code-ring": 3
    </listitem>
    <listitem>
     <para>
-     Configure container-src to sync to container-dst using the key specified
-     in the intracluster realm in
-     <literal>/etc/swift/container-sync-realms.conf</literal>
+     Configure container-src to sync to container-dst using a key specified
+     by both containers. In this example we'll use 'secret123'.
```

Comment:
Generally, we use the tag `<replaceable>` (shown in italics in the output) in these cases, for example like:

* `Replace  <replaceableable>KEY</replaceable> with your key.`
* `<screen>swift post ... -k '<replaceable>KEY</replaceable>' ...</screen>`